resourcescaler.go

```diff
@@ -171,7 +171,7 @@ func (s *AppResourceScaler) scaleServicesToZero(namespace string, serviceNames [
 		}
 	}
 
-	err = s.patchIguazioTenantAppServiceSets(namespace, jsonPatchMapper)
+	err = s.patchIguazioTenantAppServiceSets(namespace, jsonPatchMapper, false)
```

Comment:
Guessing you're planning on making this true as well pretty soon

Reply:
Nope, it will probably be a different one